kong/plugins/log-serializers/basic.lua

```diff
@@ -26,7 +28,10 @@ function _M.serialize(ngx)
       querystring = req.get_uri_args(), -- parameters, as a table
       method = req.get_method(), -- http method
       headers = req.get_headers(),
-      size = var.request_length
+      size = var.request_length,
+      tls = {
```

Comment:
this will always send an empty `tls` array even in non-tls requests. If this is okay, we leave it as is, otherwise we change it to `tls = request_tls_ver and {` (to only add the sub-table when sending TLS info

Reply:
I don't know, what do you think? Should we keep it empty or not?